<ui version="4.0" >
 <class>CopyPage</class>
 <widget class="QWizardPage" name="CopyPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>I'm copying all the images and creating the gallery. This can take some time.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QFrame" name="progressFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="progressLabel" >
        <property name="text" >
         <string>Preparing to copy...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QProgressBar" name="progressBar" >
        <property name="value" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QFrame" name="progressThumbnailFrame" >
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="imageLabel" >
        <property name="minimumSize" >
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>438</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWizardPage</class>
   <extends>QWidget</extends>
   <header location="global" >QtGui/QWizardPage</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
